include/picongpu/plugins/openPMD/Parameters.hpp

```diff
@@ -7,33 +7,12 @@ namespace picongpu::openPMD
 {
     struct PluginParameters
     {
-        std::string fileName = "simData";
+        std::string fileName = "simData"; /* Name of the openPMDSeries, excluding the extension */
```

Comment:
note: You should best comment above the line. The reason is that clang format could move the comment into the next line if the line is too long, maybe not today but at least if someone is changing e.g. the nested namespaces, ....
If the comment is in the next line you do not know anymore that it corresponds to the line above.

No need to change it in this PR, it is more of a general comment.

Reply:
I normally do that, I just copied those comments here over from the deleted lines in `ThreadParams` and they were like that